dockerfiles/docker_fedora_weekly/Dockerfile

```diff
@@ -10,19 +9,22 @@ RUN dnf -y install java java-devel unzip wget xterm python3-virtualenv && dnf cl
 
 # set java env
 ENV JAVA_HOME /usr/lib/jvm/java-openjdk
-ENV BASE_URL="https://repo1.maven.org/maven2/org/jboss/windup"
+ENV BASE_URL="https://oss.sonatype.org/content/repositories/snapshots/org/jboss/windup"
 ENV WEB_CONSOLE="mta-web-distribution"
-ENV VERSION="5.0.1.Final"
-ENV WEB_DISTRIBUTION="${WEB_CONSOLE}-${VERSION}"
-ENV WEB_CONSOLE_FILE="${WEB_DISTRIBUTION}-with-authentication.zip"
-ENV WEB_CONSOLE_FILE_PATH="${BASE_URL}/web/${WEB_CONSOLE}/${VERSION}/${WEB_CONSOLE_FILE}"
 
+# User should provide version, date, time and number of the build
+ENV VERSION=build_version
+ENV DATE_TIME_NO=build_time
 
-RUN wget -o - $WEB_CONSOLE_FILE_PATH -P /tmp| wc -l > /number && unzip -o /tmp/$WEB_CONSOLE_FILE -d /tmp
+ENV WEB_DISTRIBUTION="${WEB_CONSOLE}/${VERSION}-SNAPSHOT"
+ENV WEB_CONSOLE_FILE="${WEB_CONSOLE}-${VERSION}-${DATE_TIME_NO}-with-authentication.zip"
+ENV WEB_CONSOLE_FILE_PATH="${BASE_URL}/web/${WEB_DISTRIBUTION}/${WEB_CONSOLE_FILE}"
```

Comment:
SO this will become 
ENV WEB_CONSOLE_FILE_PATH="${BASE_URL}/web/${WEB_DISTRIBUTION}/${SNAPSHOT_BUILD}"